akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -39,6 +39,7 @@ final class ClusterSettings(val config: Config, val systemName: String) {
   private val cc = config.getConfig("akka.cluster")
 
   val LogInfo: Boolean = cc.getBoolean("log-info")
+  val LogInfoVerbose: Boolean = cc.getBoolean("log-info-verbose")
```

Comment:
This would be ok:
```
   val LogInfoVerbose: Boolean = cc.getBoolean("log-info-verbose")
   val LogInfo: Boolean = LogInfoVerbose  || cc.getBoolean("log-info")
```